### PR TITLE
update CITATION instructions: 'citEntry' -> 'bibentry'; fixes #604

### LINKED ISSUE #604: Update CITATION section
Because CRAN now says:
> Package CITATION file contains call(s) to old-style citEntry().  Please use bibentry() instead.

and our example there uses `citEntry`, so will have to be updated.

### PR DESCRIPTION
**Pull request by mpadge**

### CODE REVIEW

**Review by maelle — CHANGES_REQUESTED** [review]
I added two questions + the Spanish version is missing.

Thanks a ton!!

File: pkg_building.Rmd [hunk 1]
```diff
@@ -67,52 +67,50 @@ The [`tabulizer` package](https://github.com/ropensci/tabulizer) e.g. has an int
...
+- CRAN requires CITATION files to be declared as [`bibentry` items](https://stat.ethz.ch/R-manual/R-devel/library/utils/html/bibentry.html), and not in the previously-accepted form of [`citEntry()`](https://stat.ethz.ch/R-manual/R-devel/library/utils/html/citEntry.html).
 * If you archive each release of your GitHub repo on Zenodo, add the [Zenodo top-level DOI](https://help.zenodo.org/#versioning) to the CITATION file.
 * If one day [**after** review at rOpenSci](#authors-guide) you publish a software publication about your package, add it to the CITATION file.
 * Less related to your package itself but to what supports it: if your package wraps a particular resource such as data source or, say, statistical algorithm, remind users of how to cite that resource via e.g. `citHeader()`. [Maybe even add the reference for the resource](https://discuss.ropensci.org/t/citation-of-original-article-when-implementing-specific-methods/2312). 
 
-As an example see [nasapower CITATION file](https://github.com/ropensci/nasapower/blob/master/inst/CITATION) that refers to both the manual and a paper.
```
[the remark this answers is not in the document]
Comment (mpadge) [reply pkg_building.Rmd:74]: Adam just asked about that on slack, so he's aware. I'm going to PR to dynamite to tweak a couple of things, and let them know that they're the benchmark :smile:

File: pkg_building.Rmd [hunk 2]
```diff
@@ -67,52 +67,50 @@ The [`tabulizer` package](https://github.com/ropensci/tabulizer) e.g. has an int
...
+    key = "dynamite",
+    bibtype  = "Manual",
+    title    = "Bayesian Modeling and Causal Inference for Multivariate
+    Longitudinal Data",
+    author   = "Santtu Tikka and Jouni Helske",
+    note  = "R package version 1.0.0",
+    year     = "2022",
+    url      = "https://github.com/ropensci/dynamite"
+)
 ```
```
[the remark this answers is not in the document]
Comment (mpadge) [reply pkg_building.Rmd:112]: Yes, but all of this stuff suggests as first action `usethis::use_citation`, and that's currently wrong :scream: . so we first have to wait for that to be fixed.

[the remark this answers is not in the document]
Comment (maelle) [reply pkg_building.Rmd:112]: https://github.com/ropensci/roweb3/issues/484

File: pkg_building.es.Rmd [hunk 3]
```diff
@@ -64,50 +64,47 @@ El paquete [`tabulizer`](https://github.com/ropensci/tabulizer) por ejemplo, tie
 ## Archivo CITATION
 
 - Si tu paquete aún no tiene un archivo CITATION, puedes crear uno con `usethis::use_citation()`y llenarlo con los valores generados por la función `citation()`.
+- CRAN exige que los archivos CITATION se declaren como [`bibentry` items](https://stat.ethz.ch/R-manual/R-devel/library/utils/html/bibentry.html) y no en la forma previamente aceptada de [`citEntry()`](https://stat.ethz.ch/R-manual/R-devel/library/utils/html/citEntry.html).
```
Comment (yabellini) [inline pkg_building.es.Rmd:67]:
```suggestion
- CRAN exige que los archivos CITATION se declaren como [`bibentry` items](https://stat.ethz.ch/R-manual/R-devel/library/utils/html/bibentry.html) y no en la forma aceptada previamente de [`citEntry()`](https://stat.ethz.ch/R-manual/R-devel/library/utils/html/citEntry.html).
```

File: pkg_building.es.Rmd [hunk 4]
```diff
@@ -64,50 +64,47 @@ El paquete [`tabulizer`](https://github.com/ropensci/tabulizer) por ejemplo, tie
...
 - Si un día [**luego de** la revisión en rOpenSci](#authors-guide) publicas un artículo sobre tu paquete, añádelo al archivo CITATION.
 - Menos relacionado con tu paquete en sí mismo, pero con lo que lo soporta: si tu paquete incluye un recurso concreto, como una fuente de datos o, por ejemplo, un algoritmo estadístico, recuérdale a quien lo use cómo citar ese recurso mediante, por ejemplo, `citHeader()`. [Quizás incluso añade la referencia a ese recurso](https://discuss.ropensci.org/t/citation-of-original-article-when-implementing-specific-methods/2312).
 
-Como ejemplo, revisa el [archivo CITATION de nasapower](https://github.com/ropensci/nasapower/blob/master/inst/CITATION) que hace referencia tanto al manual como a un artículo.
-Lo único que le falta es un [DOI de Zenodo](https://discuss.ropensci.org/t/should-the-ropensci-dev-guide-include-package-citation-best-practices/1551/15) para el manual, aunque la mayoría de los usuarios probablemente acabarán citando el artículo de JOSS.
+Como ejemplo, revisa el [archivo CITATION de dynamite](https://github.com/ropensci/dynamite/blob/main/inst/CITATION) se refiere al manual R, así como a otras publicaciones asociadas.
```
Comment (yabellini) [inline pkg_building.es.Rmd:72]:
```suggestion
Como ejemplo, revisa el [archivo CITATION de dynamite](https://github.com/ropensci/dynamite/blob/main/inst/CITATION) donde se refiere al manual, así como a otras publicaciones asociadas.
```

**Review by yabellini — COMMENTED** [review]
Minor changes.

**Review by yabellini — APPROVED** [review]

**Pull request merged**